p2p/Cargo.toml

```diff
@@ -32,6 +32,10 @@ version = "0.45"
 default-features = false
 features = ["gossipsub", "identify", "mdns", "mplex", "noise", "ping", "request-response", "tcp-async-io"]
 
+[dependencies.serde]
```

Comment:
Just curious: do we have some agreement on the format of dependencies? I have seen one line form in other crates even  if both the version and the features are specified.

Reply:
I don't think we have any agreements but I've used the multi-line format in P2P when the dependency has more than one item because I find the curly bracket syntax harder to read and maintain.